app/controllers/user/team_players_controller.rb

```diff
@@ -11,41 +11,26 @@ def destroy
   end
 
   def create
-    player = Player.find_by(display_name: params[:myPlayer])
-    team = current_user.teams.find(params[:team_id])
+    player = Player.find_by(display_name: team_player_params[:myPlayer])
```

Comment:
Why is my Playercamel case?

Reply:
That param is originally in the JavaScript search bar, so it is JS convention, but then I wanted the params to match up in both spots- I could change the JS one to snake_case if you would prefer.

Reply:
I was going to ask the same thing, but I think it is actually useful to keep it like this to communicate where it's coming from